sunpy/map/sources/solo.py

```diff
@@ -0,0 +1,62 @@
+"""Solar Orbiter Map subclass definitions"""
+import astropy.units as u
+from astropy.coordinates import CartesianRepresentation
+from astropy.visualization import AsinhStretch, ImageNormalize
+
+from sunpy.coordinates import HeliocentricInertial
+from sunpy.map import GenericMap
+from sunpy.map.sources.source_type import source_stretch
+
+__all__ = ['EUIMap']
+
+
+class EUIMap(GenericMap):
+    """
+    EUI Image Map
+
+    The Extreme Ultraviolet Imager (EUI) is a remote sensing instrument onboard the
+    Solar Orbiter (SolO) spacecraft. EUI has three telescopes that image the Sun in
+    Lyman-alpha (1216 Å) and the EUV (174 Å and 304 Å). The three telescopes are the
+    Full Sun Imager (FSI) and two High Resolution Imagers (HRI). The FSI images the
+    whole Sun in both 174 Å and 304 Å. The EUV and Lyman-alpha HRI telescopes image a
+    1000"-by-1000" patch in 174 Å and 1216 Å, respectively.
+
+    References
+    ----------
+    * `Solar Orbiter Mission Page <https://sci.esa.int/web/solar-orbiter/>`_
+    * `EUI Instrument Page <https://wwwbis.sidc.be/EUI/EUI/EUI/EUI/EUI/>`_
+    * `Instrument Paper <https://doi.org/10.1051/0004-6361/201936663>`_
+    """
+
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+        self._nickname = self.detector
+        self.plot_settings['cmap'] = self._get_cmap_name()
+        self.plot_settings['norm'] = ImageNormalize(
+            stretch=source_stretch(self.meta, AsinhStretch(0.01)), clip=False)
+        self.meta['exptime'] = self.meta.get('xposure', 0.0)
```

Comment:
I think this source should override the `exposure_time` property rather than modify the header here.

Reply:
I was about to comment that that approach is inconsistent with every other map source, but now looking back I'm not sure. We seem to have a mix of overriding properties and setting metadata attributes. 

I'm not opposed to this. I agree it is a better approach.